edid_parser/tests.py

```diff
@@ -286,6 +286,27 @@ def test_descriptors(self):
         self.assertIn('Additional_Color_Point', data)
         self.assertIn('Additional_Standard_Timing', data)
 
+        del self.parser.data['Monitor_Serial_Number']
+        del self.parser.data['Monitor_Name']
```

Comment:
Why are you deleting these things here?

Reply:
Parsing the first and the second EDIDs would set these values. The EDID I'm adding to the test case also contains those. By deleting them before parsing again, we can ensure that the parser correctly recognizes those again in the third EDID.

Reply:
Yeah, I totally agree.